src/createRollupConfig.ts

```diff
@@ -15,6 +16,15 @@ import { babelPluginTsdx } from './babelPluginTsdx';
 import { TsdxOptions } from './types';
 import * as fs from 'fs-extra';
 
+const USE_COMPILER = Boolean(process.env.CLOSURE_COMPILER);
+let compilerConfig: any;
+
+// check for closure-compiler.config.js
+if (USE_COMPILER && fs.existsSync(paths.compilerConfig)) {
+  // for ex. module.exports = { compilation_level: 'ADVANCED_OPTIMIZATIONS' }
+  compilerConfig = require(paths.compilerConfig);
```

Comment:
why not just set defaults like with terser? adding a whole other config file seems suboptimal and you should be able to config via `tsdx.config.js` already